compiler-cli/src/dependencies.rs

```diff
@@ -219,73 +221,75 @@ pub fn download<Telem: Telemetry>(
     // manifest which will result in the latest versions of the dependency
     // packages being resolved (not the locked ones).
     use_manifest: UseManifest,
+    // If true we check for major version updates and print them to the console.
+    check_major_versions: bool,
```

Comment:
Create an enum instead of using bools please. `true` and `false` mean nothing without context so they are hard to understand. See the other arguments for examples.